app/views/ReasonableExcuseSelectionPage.scala.html

```diff
@@ -24,17 +25,19 @@
         link: components.link
 )
 
-@(form: Form[_], radioOptions: Seq[RadioItem])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(form: Form[_], radioOptions: Seq[RadioItem])(implicit request: UserRequest[_], messages: Messages, appConfig: AppConfig)
 
-@layout(pageTitle = messages("reasonableExcuses.title"),
+@layout(pageTitle = if(request.isAgent && !request.session.get(SessionKeys.appealType).contains(PenaltyTypeEnum.Late_Payment.toString)){
```

Comment:
Sorry to be a pain, but this would break once we have the additional penalties in, probably best to do `&& request.session.get(SessionKeys.appealType).contains(PenaltyTypeEnum.Late_Submission.toString)` instead